packages/android_alarm_manager_plus/android/src/main/java/dev/fluttercommunity/plus/androidalarmmanager/AlarmService.java

```diff
@@ -115,13 +117,15 @@ private static void scheduleAlarm(
           wakeup,
           startMillis,
           intervalMillis,
-          callbackHandle);
+          callbackHandle,
+          params);
     }
 
     // Create an Intent for the alarm and set the desired Dart callback handle.
     Intent alarm = new Intent(context, AlarmBroadcastReceiver.class);
     alarm.putExtra("id", requestCode);
     alarm.putExtra("callbackHandle", callbackHandle);
+    alarm.putExtra("params", params == null ? null : params.toString());
```

Comment:
Wouldn't this be enough?


```suggestion
    alarm.putExtra("params", params?.toString());
```

Reply:
This code written in Java ! There is no '?' command in java for null checking